test/opexecuter.h

```diff
@@ -70,14 +70,29 @@ class OpExecuter {
 
   ~OpExecuter();
 
-  void ExecuteOnNGraph();
-  void ExecuteOnTF();
-  // To Do : Overload CompareNGraphAndTF() to take in tolerance
-  void CompareNGraphAndTF();
-  void CompareNGraphAndTF(float tolerance);
-  // Executes on NGraph, executes on TF, compares the results
+  // Creates the tf graph from tf Scope
+  // Translates the tf graph to nGraph
+  // Executes the nGraph on nGraph CPU Backend
+  // Returns outputs as tf Tensors
+  void ExecuteOnNGraph(vector<Tensor>& outputs);
+
+  // Creates tf Session from tf Scope
+  // Executes on TF
+  // Returns outputs
+  void ExecuteOnTF(vector<Tensor>& outputs);
+
+  // Executes on NGraph, then executes on TF, and compares the results
   void RunTest();
 
+  // Compares NGraph and TF outputs
+  static void CompareNGraphAndTF(const vector<Tensor>& tf_outputs,
+                                 const vector<Tensor>& ngraph_outputs);
+
```

Comment:
Contemplating moving CompareNGraphAndTF function to utilities. It is comparing two vectors of Tensor (it is the context in which it is used that makes one vector tf outputs and the other ngraph outputs), and is quite independent of the OpExecuter class. Any thoughts?

```
static void CompareNGraphAndTF(const vector<Tensor>& tf_outputs,
                                 const vector<Tensor>& ngraph_outputs);
```

Reply:
Sounds like a good idea

Reply:
Yes, the functionality fits more like a utility function.

Reply:
👍